### ISSUE TITLE
BpmnPropertiesProvider for properties defined in bpmn.json

### ISSUE THREAD

**Issue by TheKetrab** [bpmn-io/bpmn-js-properties-panel#806]
__Is your feature request related to a problem? Please describe__
I am writing as my master thesis custom implementation of engine for BPMN processes. As for UI I choose bpmnio. For example I'd like to add a feature, that user defines its own script for `bpmn:ScriptTask` (via properties panel). I want to support only properties defined by [official schema](https://www.omg.org/spec/BPMN/2.0/PDF). Unfortunately default BpmnPropertiesProvider seems not to support all properties defined in bpmn-moddle's [default schema](https://github.com/bpmn-io/bpmn-moddle/blob/master/resources/bpmn/json/bpmn.json)

__Describe the solution you'd like__
I wish that BpmnPropertiesPanelModule supported all properties defined in bpmn-moddle's [default schema](https://github.com/bpmn-io/bpmn-moddle/blob/master/resources/bpmn/json/bpmn.json). I don't want to use Camunda provider because it gives access to properties from camunda namespace which I don't want to support.

__Describe alternatives you've considered__
- using Camunda gives an access to bpmn properties, but it gives access to camunda properties as well :(
- for now I solved my problem defining my own custom, very provisional provider

__Additional context__
This feature request comes from [topic](https://forum.bpmn.io/t/properties-panel-and-properties-defined-with-bpmn-moddle/8187) on bpmnio forum. Implementation would help me in my project and I am assured that in the future somebody can use the feature too.

**Comment by barmac** [comment]
Correct, we don't support editing each of the properties defined in BPMN. However, we are open to accept contributions. If you implement any of the missing properties entries, feel free to create a pull request.
